### PR TITLE
feat: allow author login/name to work in commit message templates #3793

### PR DESCRIPTION
**Pull request by djMax**
**Summary**
Fixes https://github.com/netlify/netlify-cms/issues/3793

author-name and author-login are already passed to the template, so we might as well expose them to the commit messages even if openAuthoring is false.

**Test plan**

Existing tests pass.

### CODE REVIEW

**Comment by barthc** [comment]
I suggest we also drop support for the `openAuthoring` commit template.

**Comment by djMax** [comment]
That's a big decision, which I'm happy to execute if that's what's wanted, but obviously is a breaking change.

**Comment by erezrokah** [comment]
> I suggest we also drop support for the `openAuthoring` commit template.

I specifically remember this was a requested feature, just not sure who requested it....

**Comment by barthc** [comment]
> That's a big decision, which I'm happy to execute if that's what's wanted, but obviously is a breaking change.

Agree with the breaking change, just trying to make the commit template more straight forward after this change.

**Comment by djMax** [comment]
Should I remove it from the docs as a start?

**Comment by erezrokah** [comment]
> Should I remove it from the docs as a start?
@barthc, @djMax 
Let's create a separate issue for this. Seems out of scope for this specific PR

File: packages/netlify-cms-core/src/lib/__tests__/formatters.spec.js [hunk 1]
```diff
@@ -113,6 +113,50 @@ describe('formatters', () => {
...
+            slug: 'doc-slug',
+            path: 'file-path',
+            collection,
+          },
+          true,
+        ),
+      ).toEqual(' - : Create Collection “doc-slug”');
+    });
+
+    it('should return custom update message with author information', () => {
```
Comment (erezrokah) [inline packages/netlify-cms-core/src/lib/__tests__/formatters.spec.js:137]: Just a minor issue with the test name. I tried fixing it myself but I can't push to your branch.
```suggestion
    it('should return custom create message with author information', () => {
```

**Pull request merged**